recovery/elliptics_recovery/dc_server_send.py

```diff
@@ -422,7 +422,10 @@ def _get_dest_groups(self, key_infos):
         same_meta = lambda lhs, rhs: (lhs.timestamp, lhs.size, lhs.user_flags) == (rhs.timestamp, rhs.size, rhs.user_flags)
         diff_groups = [info.group_id for info in key_infos if not same_meta(info, key_infos[0])]
 
+        corrupted_groups = [info.group_id for info in key_infos if info.flags & elliptics.record_flags.corrupted]
+
         missed_groups.extend(diff_groups)
+        missed_groups.extend(corrupted_groups)
```

Comment:
`corrupted_groups` and `diff_groups` could have some groups in common, so it is better to uniq them here.